contracts/proxy/utils/Initializable.sol

```diff
@@ -89,17 +102,20 @@ abstract contract Initializable {
      * Emits an {Initialized} event.
      */
     modifier initializer() {
-        bool isTopLevelCall = !_initializing;
-        if (!(isTopLevelCall && _initialized < 1) && !(address(this).code.length == 0 && _initialized == 1)) {
+        // solhint-disable-next-line var-name-mixedcase
```

Comment:
There are several of these for the `$` storage variable. We either keep this rule or wait on https://github.com/protofire/solhint/pull/453 to be merged if that's the case.